src/app/portlets/content-types/content-types.module.ts

```diff
@@ -1,167 +1,11 @@
+import { DotContentTypeEditResolver } from '../shared/dot-content-types-edit/dot-content-types-edit-resolver.service';
 
 @NgModule({
-    declarations: [
-        CategoriesPropertyComponent,
-        CheckboxPropertyComponent,
-        ContentTypeFieldsDropZoneComponent,
-        ContentTypeFieldsPropertiesFormComponent,
-        ContentTypeFieldsRowComponent,
-        ContentTypeFieldsTabComponent,
-        ContentTypesEditComponent,
-        ContentTypesFieldDragabbleItemComponent,
-        ContentTypesFieldsListComponent,
-        ContentTypesFormComponent,
-        ContentTypesLayoutComponent,
-        DataTypePropertyComponent,
-        DefaultValuePropertyComponent,
-        DynamicFieldPropertyDirective,
-        HintPropertyComponent,
-        NamePropertyComponent,
-        RegexCheckPropertyComponent,
-        ValuesPropertyComponent
-    ],
-    entryComponents: [
-        NamePropertyComponent,
-        CheckboxPropertyComponent,
-        CategoriesPropertyComponent,
-        DataTypePropertyComponent,
-        DefaultValuePropertyComponent,
-        HintPropertyComponent,
-        RegexCheckPropertyComponent,
-        ValuesPropertyComponent
-    ],
-    imports: [
-        ButtonModule,
-        CheckboxModule,
-        CommonModule,
-        ConfirmDialogModule,
-        ContentTypeFieldsAddRowModule,
-        ContentTypesRoutingModule,
-        DialogModule,
-        DotAddToBundleModule,
-        DotApiLinkModule,
-        DotAutofocusModule,
-        DotBaseTypeSelectorModule,
-        DotContentTypeFieldsVariablesModule,
-        DotCopyButtonModule,
-        DotDialogModule,
-        DotDirectivesModule,
-        DotIconButtonModule,
-        DotIconButtonTooltipModule,
-        DotIconModule,
-        DotMenuModule,
-        DotPageSelectorModule,
-        DotRelationshipsModule,
-        DotTextareaContentModule,
-        DotWorkflowsSelectorFieldModule,
-        DragulaModule,
-        DropdownModule,
-        DotFieldValidationMessageModule,
-        FormsModule,
-        IFrameModule,
-        InputTextModule,
-        DotListingDataTableModule,
-        MdInputTextModule,
-        MultiSelectModule,
-        OverlayPanelModule,
-        DotPushPublishContentTypesDialogModule,
-        RadioButtonModule,
-        ReactiveFormsModule,
-        SearchableDropDownModule,
-        SiteSelectorFieldModule,
-        SplitButtonModule,
-        TabViewModule,
-        DotFieldHelperModule,
-        DotMaxlengthModule,
-        DotWorkflowsActionsSelectorFieldModule,
-        DotEditToolbarModule
-    ],
-    providers: [
-        ContentTypeEditResolver,
-        DotContentTypesInfoService,
-        DotWorkflowService,
-        DragulaService,
-        FieldDragDropService,
-        FieldPropertyService,
-        FieldService,
-        FormatDateService
-    ]
+    imports: [CommonModule, ContentTypesRoutingModule, DotListingDataTableModule],
```

Comment:
Do we need `DotListingDataTableModule` here? since we have that in the listing module?